compiler/noirc_evaluator/src/brillig/brillig_gen/brillig_block.rs

```diff
@@ -101,6 +111,25 @@ impl<'block, Registers: RegisterAllocator> BrilligBlock<'block, Registers> {
         brillig_block.convert_block(dfg, call_stacks);
     }
 
+    /// Converts SSA globals into Brillig global that initializing global constants
```

Comment:
I think "that initializing global" might not belong here

Reply:
thanks for spotting this was gibberish. Switched to:
```
    /// Converts SSA globals into Brillig global values.
    ///
    /// Global values can be: 
```